lib/src/main/java/com/ibm/mobilefirstplatform/clientsdk/android/analytics/internal/BMSAnalytics.java

```diff
@@ -130,7 +131,7 @@ static public void init(Application app, String applicationName, String clientAp
             BMSAnalytics.hasUserContext = true;
```

Comment:
This was done to work around an error message that we throw in `setUserIdentity`; now that we know whether it is an initialCtx or not, we don't need this workaround of setting this variable to `true` temporarily before setting it back to false. I know it is a little convoluted, so if it doesn't make sense let me know.